src/java/htsjdk/samtools/reference/AbstractFastaSequenceFile.java

```diff
@@ -97,6 +132,35 @@ protected static File findSequenceDictionary(final File file) {
         else return null;
     }
 
+    protected static Path findSequenceDictionary(final Path file) {
```

Comment:
likewise, can the file version of findSequenceDictionary delegate to this instead?